src/FFmpeg.cpp

```diff
@@ -774,7 +774,7 @@ bool FFmpegLibs::InitLibs(wxString libpath_format, bool WXUNUSED(showerr))
    wxFileName avcodec_filename;
    wxFileName avutil_filename;
    wxFileName name(libpath_format);
-   bool gotError = false;
+   bool gotError;
```

Comment:
This one's good to go.  I bet if you looked around more, you'd find all manner of this sort of thing.